pkg/replication/admission.go

```diff
@@ -41,7 +42,8 @@ const (
 	updatedKey = "mutationTimestamp"
 	hashKey    = "mutationHash"
 
-	ReplicateKey = "replicate"
+	ReplicateKey              = "replicate"
+	ValidateKnownLocationsKey = "validateKnownLocations"
```

Comment:
For some reason all SD annotations listed above are not fully qualified (do not include `voyager.atl-paas.net` prefix), so I added a new annotation to follow established pattern.

We should probably fix this and migrate to qualified annotations at some point.
/cc @ash2k @benbarclay